meilisearch/src/lib.rs

```diff
@@ -45,6 +45,33 @@ use option::ScheduleSnapshot;
 
 use crate::error::MeilisearchHttpError;
 
+/// Default number of simultaneously opened indexes,
```

Comment:
Are these two descriptions up-to-date?

Reply:
No, good catch! Even the values were not up to date with the idea that we would use 2TiB of size!

Will update

Reply:
done

https://github.com/meilisearch/meilisearch/pull/3331/commits/234d7541348a29156418cec3a3e75562244109c2